src/core.js

```diff
@@ -315,6 +315,7 @@ const exokit = (s = '', options = {}) => {
   options.dataPath = options.dataPath || __dirname;
   options.args = options.args || {};
   options.replacements = options.replacements || {};
+  options.loadCallback = options.loadCallback || ((href, opts) => exokit.load(href, opts));
```

Comment:
In general, we want `options` to be serializable (and this is a requirement for `user-parallel` in which `window`s do not share any namespace or closures). So I don't think this is an architecture that would work going forward.